### PR TITLE
Add mandatory requirements checklist for Pull Requests (Prs)

### LINKED ISSUE #827: Adding Mandatory Requirements Checklist for Pull Requests
As part of our continuous efforts to improve code quality and contribution standards within the OpenELIS Global repository, this Issue suggests the implementation of a mandatory requirements checklist for all pull requests (PRs) submitted to the repository. The checklist aims to ensure that each PR meets certain criteria before being merged into the main codebase.

### PR DESCRIPTION
**Pull request by Agaba-derrick**
fixes #827

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -211,6 +211,13 @@ export default function App() {
                   component={() => <Home />}
                   role=""
                 />
+                <SecureRoute
```
Comment (mozzy11) [inline frontend/src/App.js:214]: you dont need this Route . 
the existing Route for `/RoutineReport` alrready handle your report

File: frontend/src/components/Reports/Routine.js [hunk 2]
```diff
@@ -98,14 +98,15 @@ export const RoutineReportsMenu = {
             "/Report?type=patient&report=haitiNonConformityByDate",
           label:  <FormattedMessage id="sideNav.label.noncomformityreportsbydate"/>,
         },
+
         {
           link:
-            config.serverBaseUrl +
-            "/Report?type=patient&report=haitiNonConformityBySectionReason",
-          label: <FormattedMessage id="sideNav.label.noncomformityreportsbyunit"/>,
+    "/RoutineReport?type=patient&report=haitiNonConformityBySectionReason",
```
Comment (mozzy11) [inline frontend/src/components/Reports/Routine.js:104]: can properly format your code

File: frontend/src/App.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ import EOrderPage from "./components/eOrder/Index";
 import RoutineIndex from "./components/Reports/routine/Index.js";
 import StudyIndex from "./components/Reports/study/index.js";
 import PrintBarcode from "./components/printBarcode/Index";
-
+import NonconformityReportsByUnit from "./components/Reports/routine/Nonconformity-Unit.js";
```
Comment (mozzy11) [inline frontend/src/App.js:39]: this import is not necesry here as its handle in the `Reports/Routine/Index.js`

**Comment by mozzy11** [comment]
Thanks @Agaba-derrick for working on this issue. 
The PR title doesnt reflect the changes in the PR

**Comment by mozzy11** [comment]
@Agaba-derrick can you also attach a video of your changes ??

**Review by mozzy11 — CHANGES_REQUESTED** [review]
See above comments

**Comment by Agaba-derrick** [comment]
> Thanks @Agaba-derrick for working on this issue. The PR title doesnt reflect the changes in the PR

hello @mozzy11 if you could give me like 1hr Iam surely resolving all this

**Pull request closed without merging**